docs/install/source-step-by-step.md

```diff
@@ -0,0 +1,344 @@
+  "property=og:title": "Install Plone backend from its Packages – Step by Step"
+  "keywords": "Plone, Plone 6, install, backend, pip, mxdev, mxmake, cookiecutter, packages, source, Zope, buildout"
+---
+
+
+(install-source-stepbystep-start-label)=
+
+# Install Plone backend from its Packages – Step by Step
+
+
+(install-source-stepbystep-backend-start-label)=
+
+## Backend
+
+For system requirements and pre-requisites for the installation see {ref}`install-source-system-requirements-label`.
+
+We install the Plone backend with `pip`, `cookiecutter-zope-instance`, `mxdev` and other fancy helpers.
+
+```{note}
+There will be one single cookiecutter template to install both backend and frontend from its packages. You will find the instructions on {ref}`install-source-installation-jump-label`. That chapter is for you if you want to develop and want to jump in with all steps prepared by an overall cookiecutter. The subsequent sections explain the installation of the backend step by step. You will learn the details of the installation included in the future overall cookiecutter.
+```
+
+
+(install-source-installation-steps-label)=
+
+### Installation steps
+
+Create a new directory to hold your project, make it your current directory, then issue the following commands in a shell session.
+
+Create a Python virtual environment in the current directory.
+
+```shell
+python3.9 -m venv venv
+source venv/bin/activate
+```
+
+Update Python package management tools.
+
+```shell
+pip install -U pip wheel
+```
+
+````{admonition} packages tree
+:class: margin toggle
+The Plone packages and dependencies are installed in trees.
+
+```
+venv/lib/Python3.x/
+site-packages/
+┌── plone/
+│   ├── …
+│   ├── app/
+│       ├── caching/
+│       ├── content/
+│       └── …
+…   
+├── zope/
+│   ├── …
+```
+````
+
+Install Plone 6 with constrained requirements using `pip`.
+
+```shell
+pip install Plone -c https://dist.plone.org/release/6.0.0a6/constraints.txt
+```
+
+````{admonition} mkwsgiinstance's minimal Zope configuration
+:class: margin toggle
+`mkwsgiinstance` creates a home with a minimal configuration for a Zope instance.
+
+```
+┌── etc/
+│   ├── site.zcml
+│   ├── zope.conf
+│   └── zope.ini
+├── inituser
+└── var/
+    ├── cache/
+    ├── log/
+    └── REAMDME.txt/
+```
+````
+
+Create a Zope instance with the given username and password in the current directory.
+
+(install-source-mkwsgiinstance)=
+
+```shell
+mkwsgiinstance -u admin:admin -d .
+```
+
+````{admonition} All commands so far
+:class: margin
+
+```shell
+python3.9 -m venv venv
+source venv/bin/activate
+pip install -U pip wheel
+pip install Plone -c https://dist.plone.org/release/6.0.0a6/constraints.txt
+mkwsgiinstance -u admin:admin -d .
+```
+````
+
+Start the Zope instance.
+
+```shell
+runwsgi ./etc/zope.ini
+```
+
+You can stop the instance later with {kbd}`ctrl-esc`.
+
+If you now open the browser with http://localhost:8080/, you see that you already can create a Plone instance.
```

Comment:
```suggestion
If you open the browser and visit http://localhost:8080/, you are presented with several choices.
```

Note that the text is not correct.

![Screen Shot 2022-06-29 at 5 20 08 AM](https://user-images.githubusercontent.com/102112/176435249-931da994-7dfc-42fe-a3db-0cea879570c7.png)

> For an introduction to Plone, success stories, demos, providers, visit [plone.com](http://plone.com/). For documentation, add-ons, support, community, visit [plone.org](http://plone.org/).

plone.com redirects to plone.org. Where should I report that issue?

Reply:
I'm going to stop here for tonight.

The next few lines confuse me and make me slam on the brakes. I open a web browser, and what exactly am I supposed to do? Click a button? Edit files? Use a cookiecutter?

Perhaps I should not visit the URL in the first place, and instead focus on configuration first?

Reply:
> plone.com redirects to plone.org. Where should I report that issue?

https://github.com/plone/Products.CMFPlone/issues - thanks!

Reply:
to reduce the confusion, I shortened the two sentences to 
_Before creating a Plone site on http://localhost:8080/, we configure our Zope instance for blobs, configure add-ons, etc.._